src/Sylius/Bundle/CoreBundle/Form/Type/Api/OrderItemType.php

```diff
@@ -28,8 +28,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         parent::buildForm($builder, $options);
 
         $builder
-            ->add('variant', 'entity_hidden', [
-                'data_class' => ProductVariant::class,
+            ->add('variant', 'sylius_product_variant_to_identifier', [
+                'class' => $options['variant_data_class'],
+                'constraints' => [
+                    new NotBlank(),
```

Comment:
I think we should add constraints on ``validation.xml`` files, as its done in whole Sylius.